dev-support/docker/Dockerfile_aarch64

```diff
@@ -130,7 +130,10 @@ ENV PATH "${PATH}:/opt/protobuf/bin"
 ####
 # Install pylint and python-dateutil
 ####
-RUN pip3 install pylint==2.6.0 python-dateutil==2.8.1
+# Disabled due to dependency availability/build issues (see HADOOP-18401)
+# instead "python" is added so the yarn-ui will build)
+# RUN pip3 install pylint==2.6.0 python-dateutil==2.8.1
```

Comment:
`mvn site` failed without this. @steveloughran 

```
$ ./dev-support/bin/create-release --docker --dockercache
...
ERROR:root:This script requires python-dateutil module to be installed. You can install it using:
         pip install python-dateutil
```

Assuming Python 3.6, `lazy-object-proxy==1.6.0` worked for me to fix the `SyntaxError: future feature annotations is not defined`.

```
RUN pip3 install pylint==2.6.0 python-dateutil==2.8.1 lazy-object-proxy==1.6.0
```

Reply:
using ayush's patch; rolling back this change. thanks for the feedback tho'